frontend/packages/dev-console/src/components/import/serverless/ServerlessSection.tsx

```diff
@@ -12,13 +12,9 @@ type ServerlessSectionProps = {
 const ServerlessSection: React.FC<ServerlessSectionProps> = ({ flags }) => {
   if (flags[FLAG_KNATIVE_SERVING]) {
     return (
-      <FormSection title="Serverless Options" divider>
```

Comment:
Why remove the divider? was this something in the designs?

Reply:
ah, I see it is a component now

Reply:
Divider being in the `FormSection` was a bad idea to begin with. The parent should control the insertion of the divider between sections.